### ISSUE TITLE
Release manifests has broken metrics

### ISSUE THREAD

**Issue by ondrejmo** [emqx/emqx-operator#168]
**Describe the bug**

Operator doesn't have correctly configured service and/or containerPort, consequently metrics endpoint doesn't work.

**To Reproduce**

```bash
kubectl apply -f https://github.com/emqx/emqx-operator/releases/download/1.1.5/emqx-operator-controller.yaml
```
`emqx-operator-controller-manager-metrics-service` has one port (https/8443), this port points to targetPort `https` on `emqx-operator-controller-manager-...` pod. The container in the pod doesn't have any such `containerPort`, because of that Prometheus metrics aren't usable.

**Expected behavior**
 
The pod should have `containerPort: 8080` exposed (according to pod logs, this is where `/metrics` actually is) and service should be adjusted accordingly (point to this port instead of 8443).

**Environment details:**:
- Kubernetes version: `1.23`
- Cloud-provider/provisioner: `k3s`
- emqx-operator version: `1.1.4`
- Install method: `static manifest`

**Comment by kiragoo** [comment]
@ondrejmo Thanks for your feedback.We'll reproduce the failure depends on `emqx-operator` 1.1.5.

**Comment by kiragoo** [comment]
@ondrejmo Hi，we will be glad for your feedback if you have any other questions for using `EMQX Operator` in `kubernetes`.😁

**Comment by kiragoo** [comment]
@ondrejmo Sorry for that, you can edit the context in `emqx-operator-controller.yaml` which releated with `emqx-operator-controller-manager-metrics-servic`:
```yaml
apiVersion: v1
kind: Service
metadata:
  labels:
    control-plane: controller-manager
  name: emqx-operator-controller-manager-metrics-service
  namespace: emqx-operator-system
spec:
  ports:
  - name: http  # https -> http
    port: 8080  # 8443 -> 8080
    targetPort: http #https -> http
  selector:
    control-plane: controller-manager
```
we will be glad for your feedback if you have any other questions.😁

**Comment by Rory-Z** [comment]
Hi, @ondrejmo we released EMQX Operator 1.1.6
In EMQX Operator 1.1.6, the pod already exposed `containerPort: 8080`
Thanks for your feedback
